server/src/main/java/com/netflix/conductor/bootstrap/BootstrapModule.java

```diff
@@ -8,6 +8,5 @@
     @Override
     protected void configure() {
         bind(Configuration.class).to(SystemPropertiesConfiguration.class);
-        bind(ModulesProvider.class);
```

Comment:
This bind wasn't doing anything, and is not needed to create an instance of ModulesProvider

`ModulesProvider modulesProvider = bootstrapInjector.getInstance(ModulesProvider.class);`